### DISCUSSION TITLE
Partisi data bertipe waktu

### DISCUSSION THREAD

**Discussion by ToniAS21** [teamalgoritma/community#168] [Q&A] [answered]
Permisi kak, saya baru ingat kemarin agak binggung membedakan kasus yang partisi data waktu yang sudah tidak menggunakan acsesor dt. apakah bisa dijelaskan bagaimana membedakannya ?

volume_googl['quarter'] = volume_googl.index.to_period('Q')   contohnya seperti ini.

Sekian terima kasih

**Comment by nisa-basalamah** [comment, marked as answer]
Halo @ToniAS21,

Untuk kasus data stock volume GOOGL: `volume_googl = stock['Volume'][['GOOGL']]`, berikut 2 hal yang perlu diperhatikan:

1. Ketika kolom tersebut sudah merupakan DatetimeIndex, maka dapat langsung menggunakan methodnya, dalam kasus ini dengan `volume_googl.index.to_period('Q')`.
2. Jika kolom tersebut masih berupa Series, maka menggunakan `.dt` accessor terlebih dahulu dan dilanjutkan dengan penggunaan methodnya. Contoh penerapannya: `volume_googl.reset_index()['Date'].dt.to_period('Q')`.

Demikian yang dapat disampaikan. Semoga bermanfaat :)
